libraries/AP_Mount/AP_Mount.cpp

```diff
@@ -206,7 +206,16 @@ const AP_Param::GroupInfo AP_Mount::var_info[] = {
 
     // 23 formerly _K_RATE
 
-    // 24 is AVAILABLE
+    // @Param: _RETRAC_ALT
+    // @DisplayName: Altitude at which to retract the mount
+    // @Description: Rectracts the mount if altitude is less than defined. Disabled when value is 0.
```

Comment:
Which altitude is this in? Absolute, above home, above terrain, etc.? I know the answer, just should be mentioned in documentation.